tests/presentations/indexable-presentation/twitter-image-test.php

```diff
@@ -127,7 +127,7 @@ public function twitter_image_open_graph_fallback_provider() {
 					],
 				],
 				'open_graph_generate_image' => 'open_graph_image.jpg',
-				'expected'                  => '',
+				'expected'                  => 'twitter_image.jpg',
```

Comment:
This test case demonstrates the desired functionality change. If the Twitter image is different than the open graph image, I think we'd want the Twitter image tag to be shown.